### ISSUE TITLE
Retrieve encoded features for test data

### ISSUE THREAD

**Issue by RenChangHow** [boyangumn/DCN-New#3]
Hi,
I'm interested in the encoded features of your model. I have a training set and a test set. I trained the model using the training set. And I wonder if there is a easy way to encode the test data using the trained model and get the encoded features (just like the model.transform() function in most sklean models) ? The provided code seems to miss this part.
Thanks.
